README.md

```diff
@@ -75,9 +75,9 @@ If you have live server extension
 
 👥 Authors 
 
-- GitHub:https://github.com/Gardimy
-- Twitter:https://twitter.com/
-- LinkedIn:https://www.linkedin.com/in/gardimy-charles-40b795212/
+- GitHub: [@githubhandle](https://github.com/Hamdullah-Andar)
+- Twitter: [@twitterhandle](https://twitter.com/HamdullahAndar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hamdullah-andar-98b35894/)
 
  🔭 Future Features 
```

Comment:
- [ ] I see that you have made a good effort on this project. However, you should add a simple report to your Pull Request as mentioned in the project requirement. See the image below.
The report should take all the points in the project requirement and provide results like the below:
- _Page titles - no issues found_
- If issues were found, they should link to the corresponding line of code in the Pull Request changes. 
Example ->  _text headings - line of code where change was made (line 34)_

![image](https://user-images.githubusercontent.com/84156714/216089785-cdd60f3a-9d3f-47f6-89d8-78f7a43e4ea6.png)